db/migrate/20190301131405_display_comments_in_ocotobox.rb

```diff
@@ -0,0 +1,13 @@
+class DisplayCommentsInOcotobox < ActiveRecord::Migration[5.2]
+  def up
+  	User.all.find_each do |user|
+  		user.update_attribute(:display_comments, true)
```

Comment:
In general we want to avoid migrations that change data using application code, because it ties the migration to a point in time.

Instead the contents of this migration should probably be:

```ruby
def change
  change_column :users, :display_comments, :boolean, default: true
end
```